UseCases/Sources/InstrumentsAppTracingInstrpkg/SpansExampleInstrument/SpansExampleInstrument/SpansExampleInstrument.instrpkg

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<package>
+	<id>org.swift.server.tracing.example.SpansExampleInstrument</id>
+	<title>SpansExampleInstrument</title>
+	<owner>
+		<name>Konrad `ktoso` Malawski</name>
+	</owner>
+
+    <!-- Instruments Developer Help: https://help.apple.com/instruments/developer/mac/current/ -->
+
+    
+    <!-- MARK: Schema Definitions -->
+    <os-signpost-interval-schema>
+        <id>tracing-spans</id>
+        <title>Example: OSSignpost tracing swift-tracing Spans</title>
+        
+        <subsystem>"org.swift.server.tracing.example"</subsystem>
+        <category>"Example"</category>
+        <name>"TracingSpans"</name>
+        
+        <start-pattern>
+            <message>"b;id:" ?id ";parent-ids:" ?parent-ids ";op-name:" ?op-name</message>
+        </start-pattern>
+        <end-pattern>
+            <message>"e;"</message>
+        </end-pattern>
+        
+        <column>
+            <mnemonic>span-id</mnemonic>
+            <title>ID</title>
+            <!-- <type>uint64</type> -->
+            <type>uint64</type>
+            <expression>?id</expression>
+        </column>
+        <column>
+            <mnemonic>parent-ids</mnemonic>
+            <title>Parent IDs</title>
+            <type>string</type>
+            <expression>?parent-ids</expression>
+        </column>
+        
+        <column>
+            <mnemonic>op-name</mnemonic>
+            <title>Operation Name</title>
+            <type>string</type>
+            <expression>?op-name</expression>
+        </column>
+    
+<!--    TODO: how to use the automatically captured thread IDs here? -->
+<!--        <column>-->
+<!--            <mnemonic>thread-start</mnemonic>-->
+<!--            <title>Thread (start)</title>-->
+<!--            <type>thread</type>-->
+<!--            <expression>?</expression>-->
+<!--        </column>-->
+<!--        <column>-->
+<!--            <mnemonic>thread-end</mnemonic>-->
+<!--            <title>Thread (end)</title>-->
+<!--            <type>thred</type>-->
+<!--            <expression>?thread</expression>-->
+<!--        </column>-->
+    </os-signpost-interval-schema>
+
+    <!-- MARK: Instrument Definitions -->
+    <instrument>
```

Comment:
just a simple instrument that shows each span (grouped by operation name) as it's own row.

This is obviously would not scale all that well in reality, but it's good for the PoC.